homeassistant/auth/__init__.py

```diff
@@ -215,12 +215,14 @@ def get_auth_mfa_module(self, module_id: str) -> Optional[MultiFactorAuthModule]
 
         return user
 
-    async def async_create_user(self, name: str) -> models.User:
+    async def async_create_user(
+        self, name: str, group_ids: Optional[List[str]] = None
+    ) -> models.User:
         """Create a user."""
         kwargs: Dict[str, Any] = {
             "name": name,
             "is_active": True,
-            "group_ids": [GROUP_ID_ADMIN],
+            "group_ids": group_ids or [GROUP_ID_ADMIN],
```

Comment:
Let's drop default admin.
```suggestion
            "group_ids": group_ids or [],
```